tests/Unit/TaskList/CompleteSetupTaskTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\tasks;
+
+use Automattic\WooCommerce\Admin\Features\OnboardingTasks\TaskLists;
+use Automattic\WooCommerce\GoogleListingsAndAds\MerchantCenter\MerchantCenterService;
+use Automattic\WooCommerce\GoogleListingsAndAds\TaskList\CompleteSetupTask;
+use Automattic\WooCommerce\GoogleListingsAndAds\Tests\Framework\UnitTest;
+use PHPUnit\Framework\MockObject\MockObject;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class CompleteSetupTaskTest
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\Tests\Unit\TaskList
+ */
+class CompleteSetupTaskTest extends UnitTest {
+
+	/** @var MerchantCenterService $merchant_center */
```

Comment:
This is a mocked object so we usually show that as:
```suggestion
	/** @var MerchantCenterService|MockObject $merchant_center */
```